Documentation/Helm-Charts/ceph-cluster-chart.md

```diff
@@ -52,24 +56,31 @@ helm install --create-namespace --namespace rook-ceph rook-ceph-cluster \
-| --------------------------------------- | ------------------------------------------------------------------------------------------ | --------------- |
-| `operatorNamespace`                     | Namespace of the Rook Operator                                                             | `rook-ceph`     |
-| `kubeVersion`                           | Optional override of the target kubernetes version                                         | ``              |
-| `configOverride`                        | Cluster ceph.conf override                                                                 | `<none>`        |
-| `toolbox.enabled`                       | Enable Ceph debugging pod deployment. See [toolbox](../Troubleshooting/ceph-toolbox.md)    | `false`         |
-| `toolbox.tolerations`                   | Toolbox tolerations                                                                        | `[]`            |
-| `toolbox.affinity`                      | Toolbox affinity                                                                           | `{}`            |
-| `toolbox.resources`                     | Toolbox resources                                                                          | see values.yaml |
-| `monitoring.enabled`                    | Enable Prometheus integration, will also create necessary RBAC rules                       | `false`         |
-| `monitoring.createPrometheusRules`      | Whether to create the Prometheus rules for Ceph alerts                                     | `false`         |
-| `monitoring.prometheusRule.labels`      | Labels applied to PrometheusRule                                                           | `false`         |
-| `monitoring.prometheusRule.annotations` | Annotations applied to PrometheusRule                                                      | `false`         |
-| `cephClusterSpec.*`                     | Cluster configuration, see below                                                           | See below       |
-| `ingress.dashboard`                     | Enable an ingress for the ceph-dashboard                                                   | `{}`            |
-| `cephBlockPools.[*]`                    | A list of CephBlockPool configurations to deploy                                           | See below       |
-| `cephFileSystems.[*]`                   | A list of CephFileSystem configurations to deploy                                          | See below       |
-| `cephObjectStores.[*]`                  | A list of CephObjectStore configurations to deploy                                         | See below       |
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| `cephBlockPools` | A list of CephBlockPool configurations to deploy | See [CephBlockPool CRD](../CRDs/Block-Storage/ceph-block-pool-crd.md) |
```

Comment:
The default column for these should have a reference to the section below, right? Instead of the separate document. The same would apply to the cephFilesystem and cephObjectStores.
```suggestion
| `cephBlockPools` | A list of CephBlockPool configurations to deploy | See [below](#ceph-block-pools) |
```

Reply:
Yes, but it would just be a little confusing when only vieweing the `values.yaml` page

Reply:
I found a compromise: I changed the table value to below, but made sure the other link is in the docs.